File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/OcsEmr/OCS2015U31GetADM3200UserInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.OcsEmr
{
    [Serializable]
    public class OCS2015U31GetADM3200UserInfo
    {
        private String _userId;
        private String _sysId;
        private String _userGroup;
        private String _userNm;

        public String UserId
        {
            get { return this._userId; }
            set { this._userId = value; }
        }

        public String SysId
        {
            get { return this._sysId; }
            set { this._sysId = value; }
        }

        public String UserGroup
        {
            get { return this._userGroup; }
            set { this._userGroup = value; }
        }

        public String UserNm
        {
            get { return this._userNm; }
            set { this._userNm = value; }
        }

        public OCS2015U31GetADM3200UserInfo() { }

        public OCS2015U31GetADM3200UserInfo(String userId, String sysId, String userGroup, String userNm)
        {
            this._userId = userId;
            this._sysId = sysId;
            this._userGroup = userGroup;
            this._userNm = userNm;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/emr/OCS2015U31GetADM3200UserInfo.java
package nta.med.data.model.ihis.emr;

public class OCS2015U31GetADM3200UserInfo {
	private String userId;
	private String sysId;
	private String userGroup;
	private String userNm;

	public OCS2015U31GetADM3200UserInfo(String userId, String sysId,
			String userGroup, String userNm) {
		super();
		this.userId = userId;
		this.sysId = sysId;
		this.userGroup = userGroup;
		this.userNm = userNm;
	}

	public String getUserId() {
		return userId;
	}

	public void setUserId(String userId) {
		this.userId = userId;
	}

	public String getSysId() {
		return sysId;
	}

	public void setSysId(String sysId) {
		this.sysId = sysId;
	}

	public String getUserGroup() {
		return userGroup;
	}

	public void setUserGroup(String userGroup) {
		this.userGroup = userGroup;
	}

	public String getUserNm() {
		return userNm;
	}

	public void setUserNm(String userNm) {
		this.userNm = userNm;
	}

}